src/rdb.c

```diff
@@ -1281,6 +1281,13 @@ int rdbSaveInfoAuxFields(rio *rdb, int rdbflags, rdbSaveInfo *rsi) {
             == -1) return -1;
     }
     if (rdbSaveAuxFieldStrInt(rdb, "aof-base", aof_base) == -1) return -1;
+
+    /* Save the active import ASM task. */
+    sds task_info = asmDumpActiveImportTask();
+    int ret = rdbSaveAuxFieldStrStr(rdb, "asm-task", task_info ? task_info : "");
```

Comment:
Just a minor thing, maybe we can call this one `cluster-asm-task` ?. Replication related ones start with `repl-`, so we can have cluster related ones with prefix `cluster-`?

Reply:
make sense, so if not in cluster mode, we may not save this info, right?